### PR TITLE
Jay

### PR DESCRIPTION
**Pull request by Jaykanathe**

### CODE REVIEW

**Comment by Jaykanathe** [comment]
product api changes

File: models/products.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        return result;
+    }
+    catch (error) {
+        logger.error(`${fileName} addProducts() ${error.message}`);
+        await dbUtil.rollback(client);
+        throw new Error(error.message);
+    }
+}
+
+// module.exports.updateProdutDetails = async (columns, values) => {
```
Comment (Sohail099) [inline models/products.js:45]: Please remove commented code

File: models/products.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    }
+}
+
+
+module.exports.addProducts = async (columns, values) => {
+    logger.info(`${fileName} addProducts() called`)
+    let sqlQuery = insertIntoTable("products", columns);
+    sqlQuery += " returning *;"
+    let client = await dbUtil.getTransaction();
+    console.log("Query : ", sqlQuery);
```
Comment (Sohail099) [inline models/products.js:32]: remove console statements like this

**Review by Sohail099 — CHANGES_REQUESTED** [review]
1. Remove console.log statements
2. Remove Commented Code
3. What about media while fetching products??

**Pull request merged**